docs/user-guide/getting-started.md

```diff
@@ -204,45 +222,57 @@ Usage: measure-engine [options]
   * -c, --context-id
       FHIR resource ID for one or more patients to evaluate.
   * -d, --data-server
-      Path to JSON configuration data for the FHIR server connection that will 
+      Path to JSON configuration data for the FHIR server connection that will
       be used to retrieve data.
     -o, --define-return-option
-      Specify define return option for evaluated define statements on measure 
-      report. Defaults to NONE. To view returned results, must specify -f 
-      JSON. 
+      Specify define return option for evaluated define statements on measure
+      report. Defaults to NONE. To view returned results, must specify -f
+      JSON.
       Default: NONE
       Possible Values: [ALL, BOOLEAN, NONE]
     --disable-retrieve-cache
       Disable the use of the retrieve cache.
+      Default: false
+    --expand-value-sets
+      By default, ValueSet resources used in CQL are expanded by the
+      terminology server. If the data server supports terminology and the
+      token :in modifier, setting this flag to false will enable use of that
+      functionality which should improve CQL engine throughput.
+      Default: true
```

Comment:
Piggybacking on my earlier comment, you may want to add `Possible Values: [true, false]` to indicate to the user that they need to explicitly write `true` or `false` to use this parameter.